src/GafferScene/LightToCamera.cpp

```diff
@@ -49,12 +49,11 @@ using namespace Gaffer;
 using namespace GafferScene;
 using namespace Imath;
 
-namespace 
+/// \todo: This stuff should all end up as part of the light accessor API once that exists
+/// In the meantime, I've tried to set it up vaguely similarly to how that will work
+namespace GoingToBeLightAccessor
```

Comment:
Let's just stick with an anonymous namespace, since LightAccessors will look different (be classes) and also we don't want any of these functions to be exported.